app/backend/src/clj/teet/task/task_commands.clj

```diff
@@ -158,8 +265,27 @@
    :project-id (project-db/task-project-id db task-id)
    :authorization {:task/review {:id task-id}}
    :pre [(task-model/waiting-for-review? (d/pull db [:task/status] task-id))]
-   :transact [{:db/id task-id
-               :task/status :task.status/reviewing}]})
+   :transact (into
+               [{:db/id task-id
+                 :task/status :task.status/reviewing}]
+               (concat (for [{id :db/id} (task-db/not-reviewed-task-files db user task-id)]
+                         {:db/id id
+                          :file/status :file.status/submitted})
+                       (for [pid (task-db/not-reviewed-file-parts db task-id)]
+                         {:db/id (first pid)
+                          :file.part/status :file.part.status/reviewing})))})
+
+(defcommand :task/start-task-part-review
+            {:doc "Start review for task part."
+             :context {:keys [db user]}
+             :payload {task-id :task-id
+                       task-part-id :taskpart-id}
+             :project-id (project-db/task-project-id db task-id)
+             :authorization {:task/review {:id task-id}}
+             :pre [(task-model/part-waiting-for-review? (d/pull db [:file.part/status] task-part-id))]
+             :transact (let [_ (taoensso.timbre/debug "TPS " task-part-id)] (into
```

Comment:
use `do` instead of let with `_` here, or remove debug log

Reply:
Unnecessary log, removed